app/views/home/_create_activity.html.erb

```diff
@@ -0,0 +1,12 @@
+<%= form_with(url: activity_index_path, method: :post) do |f| %>
+    <div class="card form-group">
+      <label for="activity-textarea">Post new activity</label>
+      <textarea  class="form-control" id="activity-textarea" name="activity[content]" rows="3" placeholder="Just start typing here..."></textarea>
+      <div style="display:none">
+        <%= f.text_field :"activity[user]" , value: current_user.id %>
```

Comment:
`current_user.id` can be placed in the controller, no extra parameter needed

Reply:
What do you mean in the controller?